src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const newSelectedTabId = selectedTabId
+  && tabs.some(tab => tab.id === selectedTabId) ? selectedTabId : tabs[0].id;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={tab.id === newSelectedTabId ? 'is-active' : ''}
```

Comment:
use classnames library for conditional classes

Reply:
it's need for all tasks like code rule